docs/en/contributor/Building-Alluxio-From-Source.md

```diff
@@ -142,7 +142,7 @@ To build against Cloudera's releases, just use a version like `$apacheRelease-cd
 
 #### Hortonworks
 
-To build against a Hortonworks release, just use a version like `$apacheRelease.$hortonRelease`
+To build against a Hortonworks release, just use a version like `$apacheRelease.$hortonworksRelease`
```

Comment:
I'm fine leaving this as is, but maybe in the future we should consider removing since hortonworks doesn't really exist any more?